posts/2024/04/29/court-systems-research.mdx

```diff
@@ -0,0 +1,42 @@
+
+Across 16 interviews and more than 10 hours of conversation, four themes emerged:
+
+- **Everyone wants eFiling systems to better help the most common court user: self-represented litigants.** Although a few courts don't offer eFiling, most have offered it for years. Many experts note that the strengths and weaknesses of the existing systems are well known, but we need a new approach to better meet the needs of the majority of court users: self-represented filers. 
+
+- **Siloed systems make it hard to streamline filing for non-lawyers.** Most existing eFiling systems present a few questions with jargon-filled dropdowns and a file upload field designed for lawyers. People without a lawyer guess what to enter, and often get it wrong. Many courts and assistance organizations try to help by providing step-by-step instructions, or a separate website that helps compile the necessary documents. These tools help, but the experience is still disjointed. And it turns out to be very hard to connect these systems together to create a single website to help non-lawyers. In sum, despite widespread eFiling, very few courts allow filers to answer questions about their problem and file a case in a single website. 
+
+- **Filer fees heavily subsidize filing systems, and that's hard on self-represented litigants.** Courts don't pay for most filing systems; the people courts serve do. Litigants have to "pay to play," and that creates an additional barrier for many self-represented applicants. Applying for a fee waiver often forces busy litigants through another set of hoops. Many of our interviewees yearned for new business models for eFiling systems that don't depend on user fees.
+
+- **Courts pay big to make small changes to systems that might help people.** Many courts and filer advocates want to change the text, flow or other details of their eFiling or case management system so it is clearer for everyone. But after the system is implemented, these changes are often very expensive. Sometimes, they even require new contracts. Even when a simple change could prevent filer mistakes, and save court timer, it often never happens.
+
+These are just a few of the findings in our report. To learn more, including a detailed overview of the way these systems fit together, read our full report.
+
+
+<p className="text-center">
+  <PurpleButton href="/pdf/Problems-Court-Filing-CMS-Perspectives-from-Advocates-and-Vendors.pdf" size="lg">Read the Report</PurpleButton>
+</p>
+
+**We are just getting started and we need your help!**
+Over the next few months, we want to talk more with courts to learn more about what could help them speed their operations and better serve their users. If you know an interested court or work at one, [please get in touch][contact-page].
+
+[contact-page]: /contact/
```

Comment:
I think this can all go under the what's next heading, where it will make more sense?